src/libYARP_OS/src/Time.cpp

```diff
@@ -81,8 +81,10 @@ static Clock *getClock()
 
 void yarp::os::impl::removeClock()
 {
-    if(pclock != nullptr)
+    if(pclock) {
         delete pclock;
+        pclock = nullptr;
```

Comment:
Thanks, good catch. Shall we use YARP_NULLPTR?

Reply:
I think now that YARP requires C++11 we can just use `nullptr` everywhere.

Reply:
Yes.. furthermore the original code used `nullptr` so I saw no reason to use `YARP_NULLPTR`